config/proto/probe.proto

```diff
@@ -61,8 +61,17 @@ message HermesProbeDef {
   cloudprober.metrics.Dist probe_latency_distribution = 7;  // Measures the latency of Hermes' probes, allowing you to diagnose a network issue or issue with Hermes.
   cloudprober.metrics.Dist api_call_latency_distribution = 8;  // Measures the latency of the API calls for the relevant storage system targeted.
 
-  // Valid values: "ns", "us" (or "µs"), "ms", "s", "m", "h".
-  string latency_unit = 9;  // default is "us"
+  enum latencyUnit {
+    LATENCY_UNIT_UNSPECIFIED = 0;
+    NS = 1;
```

Comment:
I would use more descriptive names, like "NANOS" or "NANOSECOND".

Reply:
I have changed these to their long form names.